force-app/main/default/classes/OpportunityHelper.cls

```diff
@@ -0,0 +1,6 @@
+
+public with sharing class OpportunityHelper {
+    public static Boolean isTriggerExecuted = false;
+    
+
```

Comment:
This could be class that helps the OpportunityTriggerHandler that you are going to create.